tests/reconfig/results/1.2.0/1.2.0.md

```diff
@@ -0,0 +1,87 @@
+# Reconfiguration testing Results
+
+<!-- TOC -->
+- [Reconfiguration testing Results](#reconfiguration-testing-results)
+  - [Summary](#summary)
+  - [Test environment](#test-environment)
+  - [Results Tables](#results-tables)
+    - [NGINX Reloads and Time to Ready](#nginx-reloads-and-time-to-ready)
+    - [Event Batch Processing](#event-batch-processing)
+  - [NumResources to Total Resources](#numresources-to-total-resources)
+  - [Observations](#observations)
+  - [Future Improvements](#future-improvements)
+<!-- TOC -->
+
+## Summary
+
+- Time to ready stayed consistent, if not slightly faster.
+- Reload time has slightly increased in some instances.
+- Number of batch events has reduced, subsequently increasing the size and time of each batch.
+
+## Test environment
+
+GKE cluster:
+
+- Node count: 3
+- Instance Type: e2-medium
+- k8s version: 1.27.8-gke.1067004
+- Zone: us-west2-a
+- Total vCPUs: 6
+- Total RAM: 12GB
+- Max pods per node: 110
+
+NGF deployment:
+
+- NGF version: edge - git commit 96a44240d317875406a8aef8fd1e424f2fb906eb
+- NGINX Version: 1.25.4
```

Comment:
Do we need to run it with N+ too?

Reply:
As of right now I'm not convinced of the benefit. We will still be doing reloads for N+ since these all involve Gateway resources changes, not endpoint changes.

Reply:
Because of a chance reload time might be slightly different, because we use NGINX Plus, not NGINX. so we can track those times in the future.
It might be different not because N vs N+, but because we generate slightly different config. For example https://github.com/nginxinc/nginx-gateway-fabric/blob/e1d6ebb5065bab73af3a89faba4f49c7a5b971cd/internal/mode/static/nginx/config/upstreams.go#L13-L26 . But that difference might increase in the future as we use more N+ features

Reply:
I think we need to get all of these tests automated (running both nginx and plus) and the outstanding bug fixed to truly get accurate results that we can then compare going forward.

I can run these tests again and add new tables for N+, though I'll just do 30 resources since 150 is bugged.